app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,155 @@
 <?xml version="1.0" encoding="utf-8"?>
+
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    android:id="@+id/activity_main"
+    style="@style/ConstraintStyle"
+    tools:context=".mvp.view.ActivityView">
 
     <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
+        android:id="@+id/text_result"
+        style="@style/TextViewStyle"
+        android:text="@string/text_msj"
+        app:layout_constraintBottom_toTopOf="@id/button_numb_7"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <Button
+        android:id="@+id/button_numb_7"
```

Comment:
do not use numbers in the IDs pls

Reply:
Numbers in ids were changed to name of the number.